pkg/api/v1/defaults.go

```diff
@@ -91,6 +91,10 @@ func addDefaultingFuncs() {
 			if obj.HostNetwork {
 				defaultHostNetworkPorts(&obj.Containers)
 			}
+			if obj.TerminationGracePeriodSeconds == nil {
+				period := int64(30)
```

Comment:
Please define a constant for the default value in v1/types.go (and similar for other versions).